# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/reftest.list
# Commit: ea22ccce0c4d
# Full Hash: ea22ccce0c4dbe8a1c7ace807924275497f6c899
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-25 09:36:24
# Regressor Bug: 1769963
# File Overlap Count: 2
# Description:
#   Bug 1769963 - Fix backdrop-filter inside complex transform with CSS filter chain r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147124
# ==============================================================================

diff -r 60c64f026b6b -r ea22ccce0c4d gfx/wr/wrench/reftests/filters/reftest.list
--- a/gfx/wr/wrench/reftests/filters/reftest.list	Wed May 25 05:48:08 2022 +0300
+++ b/gfx/wr/wrench/reftests/filters/reftest.list	Wed May 25 02:59:32 2022 +0000
@@ -73,6 +73,7 @@
 #platform(linux,mac) == backdrop-filter-drop-shadow.yaml backdrop-filter-drop-shadow.png
 == backdrop-filter-blur-edge-mode.yaml backdrop-filter-blur-edge-mode-ref.yaml
 == backdrop-filter-half-opacity.yaml backdrop-filter-half-opacity-ref.yaml
+platform(linux,mac) == backdrop-filter-transformed-filter.yaml backdrop-filter-transformed-filter.png
 platform(linux,max) == svg-filter-offset.yaml svg-filter-offset-ref.yaml
 skip_on(android,device) == fuzzy(1,100) svg-filter-composite.yaml svg-filter-composite-ref.yaml
 skip_on(android,device) == filter-mix-blend-scaling.yaml filter-mix-blend-scaling-ref.yaml
